src/System.Reflection.MetadataLoadContext/src/Configurations.props

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PackageConfigurations>
+      netstandard;
+      netcoreapp
+    </PackageConfigurations>
+    <BuildConfigurations>
+      netstandard;
+      netcoreapp
```

Comment:
The netcoreapp flavor should also provide all the new reflection APIs added in netcoreapp, like `Type.IsByRefLike`.

Reply:
I added https://github.com/dotnet/corefx/issues/33237 to track

Reply:
Ignore that; misunderstood comment. Making sure IsByRefLike is available for netcoreapp and tested